cg/meta/workflow/nf_analysis.py

```diff
@@ -103,3 +106,72 @@ def get_metrics_deliverables_path(self, case_id: str) -> Path:
         return Path(self.root_dir, case_id, f"{case_id}_metrics_deliverables").with_suffix(
             FileExtensions.YAML
         )
+
+    def run_analysis(
+        self,
+        case_id: str,
+        command_args: CommandArgs,
+        use_nextflow: bool,
+        dry_run: bool = False,
+    ) -> None:
+        """Execute RNAFUSION run analysis with given options."""
+        if use_nextflow:
+            self.process = Process(
+                binary=self.config.rnafusion.binary_path,
```

Comment:
We still have pipeline specific assignments here, but I suppose all these lines will be moved to the corresponding classes in the future